apps/address-book/.soliumrc.json

```diff
@@ -1,37 +1,22 @@
 {
   "extends": "solium:all",
-  "plugins": ["security"],
   "rules": {
-    "security/no-low-level-calls": "off",
-    "security/no-inline-assembly": "off",
-    "emit": "off",
-    "error-reason": "off",
-    "imports-on-top": "error",
-    "variable-declarations": "error",
-    "array-declarations": "error",
-    "operator-whitespace": "error",
-    "conditionals-whitespace": "error",
-    "comma-whitespace": "error",
-    "semicolon-whitespace": "error",
-    "function-whitespace": "error",
-    "lbrace": "error",
-    "mixedcase": "error",
-    "camelcase": "error",
-    "uppercase": "error",
-    "no-empty-blocks": "error",
-    "no-unused-vars": "error",
-    "quotes": "error",
-    "blank-lines": "error",
-    "indentation": "error",
+    "imports-on-top": ["error"],
+    "variable-declarations": ["error"],
+    "array-declarations": ["error"],
+    "operator-whitespace": ["error"],
+    "lbrace": ["error"],
+    "mixedcase": 0,
+    "camelcase": ["error"],
+    "uppercase": 0,
+    "no-empty-blocks": ["error"],
+    "no-unused-vars": ["error"],
+    "quotes": ["error"],
+    "indentation": 0,
+    "whitespace": ["error"],
+    "deprecated-suicide": ["error"],
     "arg-overflow": ["error", 8],
-    "whitespace": "error",
-    "deprecated-suicide": "error",
-    "pragma-on-top": "error",
-    "function-order": "error",
-    "no-constant": "error",
-    "value-in-payable": "error",
-    "max-len": "error",
-    "visibility-first": "error",
-    "linebreak-style": "error"
+    "pragma-on-top": ["error"],
+    "security/enforce-explicit-visibility": ["error"]
```

Comment:
Do we want to upgrade this right now, or was this part of the upgrade to `ethlint`, which we have now delayed?